### DISCUSSION TITLE
I cannot change any settings from hidden pages, open source?

### DISCUSSION THREAD

**Discussion by miguelgargallo** [supabase/supabase#27396] [Feature Requests]
When are you going to allow users to access this functionalities coded? when

https://myopensourceurl .app/new/default-org-slug
![image](https://github.com/supabase/supabase/assets/5947268/330a76f0-23d9-43fd-8413-a9202a923a75)
https://myopensourceurl .app/project/default/auth/templates
![image](https://github.com/supabase/supabase/assets/5947268/34dc2275-ef1e-4f57-b366-f87508f1b132)
https://myopensourceurl .app/new/default-org-slug
![image](https://github.com/supabase/supabase/assets/5947268/22e2e220-5950-4cb5-98c0-7c7b87d5edb8)
https://myopensourceurl .teide.app/project/default/auth/providers

**Comment by ryunosuke21** [comment]
I agree 100%, I've been so confused on how to do certain things while self hosting. I think better documentation would be a huge step for this issue to be solved.

For anyone who's been having the same issues:

### Here are some breadcrumbs I've managed to find:

- **Auth Providers:** check out [this issue](https://github.com/orgs/supabase/discussions/4885). They added some lines to the docker-compose.yml and the .env file. [This article](https://medium.com/@christopher.mathews/how-to-self-host-supabase-on-your-own-computer-full-guide-cceac9b740ea#a2af) also provides good insight on what to do.
- **Email Templates:** apparently its the same thing about changing stuff in the docker-compose.yml. I'm trying to find any relevant links but so far I haven't been able to so far. _Edit: Found the link, its in [this discord discussion](https://discord.com/channels/839993398554656828/1222621691256111114/1222621691256111114)_

Also, [this video](https://www.youtube.com/watch?v=wyUr_U6Cma4&t=1389s) by [activenode](https://www.youtube.com/@activenode) is really helpful for some things, but as far as I understand, he's using 2 servers, one for s3 buckets and one for supabase. If you're not using s3 buckets this would still work.

**Comment by charislam** [comment]
We definitely hear you about improving documentation for local dev and self-hosting. Regarding email templates, you can customize them here: https://supabase.com/docs/guides/cli/customizing-email-templates

**Comment by miguelgargallo** [comment]
Begin by putting here the hidden pages on the open source version, sidebar and also under policies

![image](https://github.com/supabase/supabase/assets/5947268/99af0d48-363b-42b2-a681-70e0704e02c4)

**Comment by encima** [comment]
@miguelgargallo those pages are hidden because they are not all functional. Self-hosted Supabase does not use the Management API and pages are not shown because they do not fully function. 

We will leave this issue open as it can help us track the improvement of documentation of the current functionality and setup processes

**Comment by miguelgargallo** [comment]
No @encima and supabase team, pages are coded, functionalities exist, but because of a commercial thing, I thought this is hidden with the goal to push users to the commercial and not opensource

![image](https://github.com/supabase/supabase/assets/5947268/633bc91b-ef3b-4942-b7c6-a8010d0e8c46)

this is a screenshot from my opensource version, not even 2!

**Comment by encima** [comment]
@miguelgargallo I understand why you may think that and it is certainly not an uncommon practice. 
However, this is not the case here. Pages are not enabled when we cannot guarantee the same experience as we can in the hosted version. 
I am not sure what your screenshot is showing but maybe I have missed some context here?

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions/enhancements.

**Comment by opxdelwin** [comment]
Hey @encima, I'd love to work on this to implement the screen with the available auth providers as of now. 

I understand that the team would love to launch this once _Everything_ is functional, but let the self hosted users access what's available, and we can expand the choices from there.

**Reply by encima** [reply]
You are very welcome to open a PR to expose settings that are available now!

**Reply by opxdelwin** [reply]
Cool, could you please confirm on the available providers for a reference?
